skytap/version.go

```diff
@@ -0,0 +1,9 @@
+package skytap
+
+const userAgentVersion = `{
```

Comment:
Not sure if necessary to have the user agent as String and then decoded as this isn't adding any value as it is.